pyproject.toml

```diff
@@ -129,3 +128,106 @@ web-node = ["gunicorn", "honcho"]
+
+[tool.poetry.plugins."invenio_base.apps"]
+inspire_arxiv = "inspirehep.modules.arxiv:InspireArXiv"
+inspire_authors = "inspirehep.modules.authors:InspireAuthors"
+inspire_crossref = "inspirehep.modules.crossref:InspireCrossref"
+inspire_disambiguation = "inspirehep.modules.disambiguation:InspireDisambiguation"
+inspire_fixtures = "inspirehep.modules.fixtures:InspireFixtures"
+inspire_forms = "inspirehep.modules.forms:InspireForms"
+inspire_literaturesuggest = "inspirehep.modules.literaturesuggest:InspireLiteratureSuggest"
+inspire_migrator = "inspirehep.modules.migrator:InspireMigrator"
+inspire_orcid = "inspirehep.modules.orcid:InspireOrcid"
+inspire_records = "inspirehep.modules.records.ext:InspireRecords"
+inspire_search = "inspirehep.modules.search:InspireSearch"
+inspire_theme = "inspirehep.modules.theme:INSPIRETheme"
+inspire_tools = "inspirehep.modules.tools:InspireTools"
+inspire_utils = "inspirehep.utils.ext:INSPIREUtils"
+inspire_workflows = "inspirehep.modules.workflows:InspireWorkflows"
+
+[tool.poetry.plugins."invenio_base.blueprints"]
+inspirehep_editor = "inspirehep.modules.editor:blueprint"
+
+[tool.poetry.plugins."invenio_celery.tasks"]
+inspire_migrator = "inspirehep.modules.migrator.tasks"
+inspire_orcid = "inspirehep.modules.orcid.tasks"
+inspire_records = "inspirehep.modules.records.tasks"
+inspire_refextract = "inspirehep.modules.refextract.tasks"
+
+[tool.poetry.plugins."invenio_db.alembic"]
+inspirehep = "inspirehep:alembic"
+
+[tool.poetry.plugins."invenio_db.models"]
+inspire_workflows_audit = "inspirehep.modules.workflows.modules"
+
+[tool.poetry.plugins."invenio_jsonschemas.schemas"]
+inspire_records = "inspire_schemas"
+
+[tool.poetry.plugins."invenio_pidstore.fetchers"]
+inspire_recid_fetcher = "inspirehep.modules.pidstore.fetchers:inspire_recid_fetcher"
+
+[tool.poetry.plugins."invenio_pidstore.minters"]
+inspire_recid_minter = "inspirehep.modules.pidstore.minters:inspire_recid_minter"
+
+[tool.poetry.plugins."invenio_search.mappings"]
+holdingpen = "inspirehep.modules.workflows.mappings"
+
+[tool.poetry.plugins."invenio_workflows.workflows"]
+article = "inspirehep.modules.workflows.workflows:Article"
+author = "inspirehep.modules.workflows.workflows:Author"
+manual_merge = "inspirehep.modules.workflows.workflows:ManualMerge"
+edit_article = "inspirehep.modules.workflows.workflows:EditArticle"
+core_selection = "inspirehep.modules.workflows.workflows:CoreSelection"
+
+[tool.poetry.plugins."invenio_workflows_ui.actions"]
+author_approval = "inspirehep.modules.workflows.actions.author_approval:AuthorApproval"
+hep_approval = "inspirehep.modules.workflows.actions.hep_approval:HEPApproval"
+merge_approval = "inspirehep.modules.workflows.actions.merge_approval:MergeApproval"
+match_approval = "inspirehep.modules.workflows.actions.match_approval:MatchApproval"
+
+[build-system]
+requires = ["setuptools >= 40.6.0", "wheel"]
+build-backend = "setuptools.build_meta"
```

Comment:
not sure this will work. If not, you can explicitly do `pip install setuptools` in the Dockerfile.